crates/bitwarden/src/crypto/master_key.rs

```diff
@@ -0,0 +1,140 @@
+//! Cryptographic primitives used in the SDK
+
+#[cfg(feature = "internal")]
```

Comment:
This file is already defined as this in crypto/mod.rs:
```
#[cfg(feature = "internal")]
mod master_key;
```

So any cfg internal in this file is not necessary now.